neurodamus/node.py

```diff
@@ -502,7 +554,12 @@ def create_synapses(self):
             self._load_projections(pname, projection)
 
         if SimConfig.dry_run:
-            self._collect_display_syn_counts(synapse_counter)
+            self.syn_total_memory = self._collect_display_syn_counts(synapse_counter)
+            if MPI.rank == 0:
+                total_memory = self.cells_total_memory + self.syn_total_memory/1024
+                + self.avg_tasks_usage_mb*MPI.size
```

Comment:
Maybe fix this one. The `+ self.avg_tasks_usage_mb*MPI.size` is an independent line, not part of the expression on line 559

Reply:
Thanks for spotting this! I've fixed it.